cypress/plugins.js

```diff
@@ -8,19 +7,15 @@ const { getWebpackConfig } = require('nuxt')
  */
 module.exports = (on, config) => {
   on('dev-server:start', async (options) => {
-    let webpackConfig = await getWebpackConfig('client', {
+    const webpackConfig = await getWebpackConfig('client', {
       for: 'dev',
     })
 
```

Comment:
With webpack-dev-server 1.1.3 we do not need those shenanigans anymore ;)

Woohoo !! \o/